text/0000-experimental-features.md

```diff
@@ -0,0 +1,78 @@
+# Meta
+[meta]: #meta
+- Name: Space and Process for Experimental Features
+- Start Date: 2020-07-02
+- Author(s): [@jkutner](https://github.com/jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for allowing experimental features to enter Pack and Lifecycle without requiring a specification, core team review, or RFC.
+
+# Motivation
+[motivation]: #motivation
+
+End-users and early adopters of CNB have real problems that need to be solved by the CNB project. Sometimes these problems prevent them from adopting CNB in their organizations. Our RFC process, specification guidelines, and governance structure provide a strong mechanism through which we can enact change and solve problems; but sometimes it's too heavy and inhibits experimentation.
+
+We need a way to introduce experiments before writing an RFC. Such a mechanism would allow us to deliver new capabilities faster, improve our means of collecting real end-user feedback, and improve the quality of our RFCs.
+
+# What it is
+[what-it-is]: #what-it-is
+
+- *experimental feature* - any behavior or options that are likely to change before being added to the specification
+
+This proposal affects buildpack users, platform implementors, and project contributors who want to introduce experimental features into Pack or Lifecycle. Today, any significant change that impacts end-users is introduced through an RFC, which requires a successful vote by the core team before being introduced into the specification (or the Pack platform).
```

Comment:
I wonder if this particular mechanism for enabling experimental features is better suited to `pack` than the lifecycle. Not all lifecycle features can or should be enabled with flags (think buildpack API features).

I am wondering if experimental API versions as proposed in https://github.com/buildpacks/rfcs/pull/92 would better support this goal at the lifecycle level